examples/triangle/triangle.cpp

```diff
@@ -126,6 +126,9 @@ class VulkanExample : public VulkanExampleBase
 
 	~VulkanExample()
 	{
+		// SRS - Ensure all operations on the device have finished before destroying resources
+		vkDeviceWaitIdle(device);
```

Comment:
This shouldn't be necessary. The base class already does a `vkDeviceWaitIdle` at the end of the render loop.

Reply:
See comment above - same issue and same solution.